src/main/java/Parser.java

```diff
@@ -0,0 +1,97 @@
+public class Parser {
+
+  public Parser() { //parser object
+  }
+
+  /***
+   * Returns a string array containing the command type, command details and date details. If there
+   * are no details then the array slot is left as null. Reads the raw input from the user as
+   * a string and split the based on whitespace
+   * @param String input from the user
```

Comment:
I think you have to leave a line between the description and the @params section?

Reply:
Done